deploy/README.md

```diff
@@ -192,15 +192,16 @@ WBS Deploy may also receive minor and patch updates, but, as noted above, they a
 
 #### Minor and patch updates for WBS images
 
-Because WBS Deploy always references the latest minor and patch releases of compatible WBS images, non-breaking changes (including security updates) are applied automatically when re-creating Docker containers.
+Because WBS Deploy always references the latest minor and patch releases of compatible WBS images, non-breaking changes (including security updates) can be pulled at any time.
 
 This is always safe to do. Simply run:
 
 ```sh
 docker compose down
+docker compose pull
```

Comment:
`docker compose up --pull always`

For this section I think there isn't value to list both the down/up and down/"up --pull always" options as the section is exclusively about how to update.

Reply:
I do not agree. I think it is nice to have the pull as an explicit step here.

Your one liner is mentioned below as a hint too.

Reply:
I see as unnecessary detail that can be distracting, but if you think it is better then I'm fine leaving it.

Reply:
I'll go ahead and merge it now in order to have the actual fix available. The updating docs need some more general overhaul anyway as far as I am concerned. Let's consider the different options with the bigger context again.